gozw.go

```diff
@@ -190,6 +190,38 @@ func (c *Client) Nodes() map[byte]*Node {
 	return c.nodes
 }
 
+func contains(s []byte, e byte) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
+	}
+	return false
+}
+
+// Will prune any duplicate nodes from the database
+func (c *Client) PruneNodes() error {
+	dbNodes := c.Nodes()
+	initData, err := c.serialAPI.GetInitAppData()
+	nodeList := initData.GetNodeIDs()
+
+	if err != nil {
+		fmt.Print(err)
```

Comment:
If zap is available that would be preferable -- do you know what a common cause for this failure would be?